pkg/kritis/kubectl/plugins/resolve/resolve_test.go

```diff
@@ -255,9 +255,12 @@ spec:
 	testutil.CheckErrorAndDeepEqual(t, false, err, expected, actual)
 }
 
-func formatTestYaml1() yaml.MapSlice {
+func formatTestYaml1(t *testing.T) yaml.MapSlice {
 	m := yaml.MapSlice{}
```

Comment:
Should we throw in a t.Helper() call now that this is getting a t passed in?

Reply:
i think we shd do that to all the testutil finctions. 
When debugging tests, i  always had to guess the line number where the test failed.
We should definitely call this function everwhere else.